cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,249 @@ Cask Data Application Platform Release Notes
+
+- :cask-issue:`CDAP-12283` - Removed version for artifacts for authorization policy to be consistent with applications. From 4.3 onwards CDAP does not support policies on artifact/application version. 
+- :cask-issue:`CDAP-12298` - Individual system service status API no longer has to goes through CDAP master.
+
+- :cask-issue:`CDAP-8629` - Allow programs to have concurrent runs in integration test cases.
+
+- :cask-issue:`CDAP-9224` - Adds upgrade step when importing a pipeline configuration
+
+- :cask-issue:`CDAP-9953` - Removed dataset usage in the Hive source and sink, which allows it to work in Spark and fixes a race condition that could cause pipelines to fail with a transaction conflict exception.
+
+- :cask-issue:`CDAP-10228` - Sinks in streaming pipelines no longer have their prepareRun and onFinish methods called if the rdd for that batch is empty
+
+- :cask-issue:`CDAP-11704` - Fixed CDAP to work with and publish to Yarn Timeline Server in a secure environment.
+
+- :cask-issue:`CDAP-11783` - HBaseDDLExecutor implementation is now localized to the containers without adding it in the container classpath.
+
+- :cask-issue:`CDAP-11800` - Fixed a bug that the stream client gave wrong error message when the authorization check failed for stream read.
+
+- :cask-issue:`CDAP-11880` - Fixed a bug that hydrator pipelines and other programs do not create datasets at runtime with correct impersonated user.
+
+- :cask-issue:`CDAP-11884` - Allowing dots in plug in/artifact names
+
+- :cask-issue:`CDAP-11944` - Removed non-configurable properties from CSD/Ambari
+
+- :cask-issue:`CDAP-11948` - Fix a bug where committed data could be removed during HBase table flush or compaction.
+
+- :cask-issue:`CDAP-11955` - Fixed a bug that sometimes wrong user is used in explore, which results in the failure of deleting namespace.
+
+- :cask-issue:`CDAP-11985` - Fixed a bug that ugi provider returns the old and incorrect ugi information.
+
+- :cask-issue:`CDAP-12054` - Fixed PartitionedFileSet to work with CombineFileInputFormat, as input to a batch job.
+
+- :cask-issue:`CDAP-12122` - Fixed a bug in the pipeline planner that caused some pipelines to fail to deploy with a NoSuchElementException
+
+- :cask-issue:`CDAP-12125` - Fixed a bug in mapreduce pipeline timing metrics, where time for a stage could include time spent in other stages.
+
+- :cask-issue:`CDAP-12130` - Fixed an issue that will enable send-to-directive working on derived columns in data prep.
+
+- :cask-issue:`CDAP-12161` - Fixed a bug in StructuredRecord where a union of null and at least two other types could not be set to a null value.
+
+- :cask-issue:`CDAP-12170` - Fixed a bug where committed files of a PartitionedFileSet could be removed during transaction rollback in the case PartitionOutput#addPartition was called for a partition that already e
```

Comment:
This is incomplete. Please update to:

'Fixed a bug where committed files of a PartitionedFileSet could be removed during transaction rollback in the case PartitionOutput#addPartition was called for a partition that already existed. 
With this fix, PartitionedFileSet#getPartitionOutput should now only be called within a transaction.'